### PR TITLE
[BEAM-2161] support string operators

### PR DESCRIPTION
**Pull request by xumingming**
https://issues.apache.org/jira/browse/BEAM-2161 @XuMingmin

### CODE REVIEW

File: dsls/sql/src/test/java/org/apache/beam/dsls/sql/interpreter/operator/string/BeamSqlPositionExpressionTest.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import java.util.List;
+
+import org.apache.beam.dsls.sql.interpreter.BeamSQLFnExecutorTestBase;
+import org.apache.beam.dsls.sql.interpreter.operator.BeamSqlExpression;
+import org.apache.beam.dsls.sql.interpreter.operator.BeamSqlPrimitive;
+import org.apache.calcite.sql.type.SqlTypeName;
+import org.junit.Test;
+
+/**
+ * Created by xumingmingv on 5/5/17.
```
Comment (XuMingmin) [inline dsls/sql/src/test/java/org/apache/beam/dsls/sql/interpreter/operator/string/BeamSqlPositionExpressionTest.java]: pls update JavaDoc here

**Comment by XuMingmin** [comment]
overall LGTM,

**Comment by jbonofre** [comment]
R: @jbonofre

**Comment by xumingming** [comment]
Conflict resolved.

**Comment by jbonofre** [comment]
Thanks, waiting the end of Jenkins before merging.

**Comment by jbonofre** [comment]
Merged on `DSL_SQL` branch. You can close this PR. Thanks !

**Pull request closed without merging**